.github/pull_request_template.md

```diff
@@ -0,0 +1,26 @@
+Closes <!-- issue reference -->
+
+## Type of change
+
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Bug fix (non-breaking change which fixes a bug)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] Refactoring (only inner code structure was changed)
+- [ ] Documentation update (no code change was made)
+- [ ] CI/CD
+- [ ] Miscellaneous
+
+## What I have made
+
+<!-- write down what changes have been made, what was removed/added -->
+<!-- e.g. I added a new route called x, it has the parameters y and z -->
+
+## Checklist
+
+Either tick or cross out the items that do not apply (using \~\~example text\~\~) and give a reason why the item does not apply.
```

Comment:
Is there a reason for having to cross something out instead of just leaving it unticked?

Reply:
Ok with me 👍